docs/extensions/community.md

```diff
@@ -0,0 +1,9 @@
+# Community Extensions
+
+This file should have all the known Plates extensions listed.
+
+Please feel free submit a [Pull Request](https://github.com/thephpleague/plates) to add your extension to this list.
+
+ - [Laravel Provider](https://github.com/franzliedke/laravel-plates)
+ - [Attributes Rendering](https://github.com/RobinDev/platesAttributes) - Permits to transform an array in html tag attributes.
```

Comment:
Change the description to: `Transforms arrays into html tag attributes.`

Reply:
I took the description from the repository. I will change BTW.